sktime/annotation/datagen.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+"""Synthetic data generating functions."""
+
+from typing import Union
+
+import numpy as np
+import numpy.typing as npt
+from sklearn.utils.validation import check_random_state
+
+# what if we could pass data_gen functions - rather than assuming means?
+#   - could make it easier to swap out different mean assumptions
```

Comment:
Wondering - is it worth leaving this comment/commented code at the bottom here?  Could be cleaner to remove it and just add ideas elsewhere?  But can also see the value of leaving some ideas in the code as well.  If the latter though maybe  we could make it more organized?

Reply:
... and cleaned up the comments :wink: